sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/ordering.scala

```diff
@@ -19,18 +19,28 @@ package org.apache.spark.sql.catalyst.expressions
 
 import org.apache.spark.sql.catalyst.InternalRow
 import org.apache.spark.sql.catalyst.expressions.BindReferences.bindReferences
+import org.apache.spark.sql.catalyst.expressions.codegen.GenerateOrdering
 import org.apache.spark.sql.types._
 
 
+/**
+ * Inherits some default implementation for Java from `Ordering[Row]`
+ */
+class BaseOrdering extends Ordering[InternalRow] {
+  def compare(a: InternalRow, b: InternalRow): Int = {
+    throw new UnsupportedOperationException
```

Comment:
do we still need to default implementation?

Reply:
Ah, that's right. I removed.